src/components/panel/panel.spec.js

```diff
@@ -189,6 +190,40 @@ describe('$mdPanel', function() {
       expect(panelRef.isAttached).toEqual(false);
     });
 
+    fit('should resolve on animate failure when opening', function () {
+      var openRejected = false;
+
+      panelRef._animateOpen = function() {
+        return panelRef._$q.reject();
+      };
+
+      panelRef.open().then(function() {}, function () {
```

Comment:
Can you make sure that the openResolve is called? Implicitly this is what you are testing right now, but having both makes it more explicit.